RHEL/7/profiles/stig-rhel7-disa.xml

```diff
@@ -44,7 +44,7 @@ Storage deployments.
 <refine-value idref="var_password_pam_retry" selector="3" />
 <refine-value idref="var_accounts_max_concurrent_login_sessions" selector="10" />
 <refine-value idref="var_accounts_tmout" selector="10_min" />
-<refine-value idref="var_time_service_set_maxpoll" selector="disa" />
+<refine-value idref="var_time_service_set_maxpoll" selector="17_seconds" />
```

Comment:
DISA updated their content to match the system default (10 seconds). This should be ``system_default``